flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/parse/ClearOperationParseStrategy.java

```diff
@@ -29,7 +29,7 @@
     static final ClearOperationParseStrategy INSTANCE = new ClearOperationParseStrategy();
 
     private ClearOperationParseStrategy() {
-        super(Pattern.compile("CLEAR;?", DEFAULT_PATTERN_FLAGS));
+        super(Pattern.compile("CLEAR(\\s*;)?", DEFAULT_PATTERN_FLAGS));
```

Comment:
What about just using `CLEAR\s*;?`. This way you check any eventual spaces, but not a space after the `;` (which should be the beginning of another command)

Reply:
Same comments for all the ones below

Reply:
done, thanks